clang/lib/CIR/CodeGen/CIRGenBuiltinX86.cpp

```diff
@@ -68,6 +68,27 @@ static mlir::Value emitVectorFCmp(CIRGenBuilderTy &builder,
   return bitCast;
 }
 
+// Convert the mask from an integer type to a vector of i1.
+static mlir::Value getMaskVecValue(CIRGenFunction &cgf, mlir::Value mask,
```

Comment:
This function is also being upstreamed in https://github.com/llvm/llvm-project/pull/168591 which I expect to merge today. You'll need to rebase this PR after that one is merged.